### ISSUE TITLE
obfsproxy Support ?

### ISSUE THREAD

**Issue by Mehran** [Mi-Al/OpenVPNassistent#1]
is it support obfsproxy ?   (obfs3proxy/obfs4proxy)

**Comment by Mi-Al** [comment]
No, right now it is a small script to help to deploy OpenVPN. It generates comprehensive configuration files with embedded keys for the server and clients simultaneously. Configs with embedded keys can be delivered to clients pretty easy. So the script lets to deploy OpenVPN or update all configs very fast. Workable, but rather still draft.
